thebeetoken/beenest-web
# Checkbox → Switch in /host/listings

## Description
Display the "Publish" control as a switch instead of a checkbox, which is more consistent with the original design documents.

Also adds a `title` to display instructions to complete listing on hover when this control is disabled

## How to Test
Verify aesthetics and check for regressions relative to instructions on #26 

Which devices did you test on?

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
The messaging is fairly ad hoc; we may want to revise in the future.

There are likely to be merge conflicts with #40, so one of these PRs will need conflict resolution depending on which we merge first.

## Learnings
Many React components [expect use of `css-loader`](https://github.com/gfazioli/react-toggle#default-styles) or similar
